lib/go-tc/enum.go

```diff
@@ -53,6 +53,8 @@ type DeliveryServiceName string
 // CacheType is the type (or tier) of a CDN cache.
 type CacheType string
 
+const MonitorTypeName = "RASCAL"
```

Comment:
Do we think this should be a config?  I have seen several times now where the type "RASCAL" was hard-coded and we had to change our TM type.  IMO you should be able to make the type whatever you want and it should work.

Reply:
I think it should definitely be a config, once we've moved everything to Go. But right now, it's still hard-coded all over Perl. 

I don't think we should make a config yet, which would make people think they could change it, when it'd still break to be anything but "RASCAL".

But definitely +100 when it's all Go, and we can set one variable. I've wanted to change our db "RASCAL" type to something like "monitor" for ages.